src/get_functions.jl

```diff
@@ -43,6 +52,15 @@ function get_spacing(vtk_file)
   return spacing
 end
 
+"""
+    get_spacing(pvtk_file::PVTKFile)
+
+Retrieve a vector with the regular increments in each coordinate direction of the uniform grid from the given [`PVTKFile`](@ref) file.
+
+See also: [`PVTKFile`](@ref)
+"""
+get_spacing(pvtk_file::PVTKFile) = get_spacing(VTKFile(pvtk_file.saved_files[1]))
+
 
 """
     get_whole_extent(vtk_file)
```

Comment:
```suggestion
    get_whole_extent(vtk_file::VTKFile)
```
To be consistent with the code.